sdk/python/foundation-models/system/content-safety/deploy_llama_with_content_safety.ipynb

```diff
@@ -0,0 +1,503 @@
+    "print(f\"AACS ResourceId is {aacs_resource_id}\")\n",
+    "\n",
+    "aacs_access_key = acs_client.accounts.list_keys(\n",
+    "    resource_group_name=resource_group, account_name=aacs.name\n",
+    ").key1\n",
+    "print(f\"AACS access key is {aacs_access_key}\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 3. Create Azure AI Content Safety enabled LLaMA online endpoint"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3.1 Check if LLaMA model is available in the AML registry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "reg_client = MLClient(\n",
+    "    credential,\n",
+    "    subscription_id=subscription_id,\n",
+    "    resource_group_name=resource_group,\n",
+    "    registry_name=registry_name,\n",
+    ")\n",
+    "version_list = list(\n",
+    "    reg_client.models.list(model_name)\n",
+    ")  # list available versions of the model\n",
+    "llama_model = None\n",
+    "if len(version_list) == 0:\n",
+    "    print(\"Model not found in registry\")\n",
+    "else:\n",
+    "    model_version = version_list[0].version\n",
+    "    llama_model = reg_client.models.get(model_name, model_version)\n",
+    "    print(\n",
+    "        f\"Using model name: {llama_model.name}, version: {llama_model.version}, id: {llama_model.id} for inferencing\"\n",
+    "    )"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### 3.2 Create LLaMA online endpoint\n",
+    "This step may take a few minutes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import ManagedOnlineEndpoint\n",
+    "\n",
+    "# Check if the endpoint already exists in the workspace\n",
+    "try:\n",
+    "    endpoint = ml_client.online_endpoints.get(endpoint_name)\n",
+    "    print(\"---Endpoint already exists---\")\n",
+    "except:\n",
+    "    # Create an online endpoint if it doesn't exist\n",
+    "\n",
+    "    # Define the endpoint\n",
+    "    endpoint = ManagedOnlineEndpoint(\n",
+    "        name=endpoint_name, description=\"Test endpoint for model\"\n",
+    "    )\n",
+    "\n",
+    "    # Trigger the endpoint creation\n",
+    "    try:\n",
+    "        ml_client.begin_create_or_update(endpoint).wait()\n",
+    "        print(\"\\n---Endpoint created successfully---\\n\")\n",
+    "    except Exception as err:\n",
+    "        raise RuntimeError(\n",
+    "            f\"Endpoint creation failed. Detailed Response:\\n{err}\"\n",
+    "        ) from err"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### 3.3 Create environment for LLaMA endpoint\n"
```

Comment:
Leveraging model package will make these steps of creating env, deploying with custom score.py, etc. much simpler

Reply:
will send a separate pr, working with training team to solve it.